docs/config/README.md

```diff
@@ -340,6 +340,28 @@ variable = "SHELL"
 default = "unknown shell"
 ```
 
+## Elixir
```

Comment:
I believe there's also a `Default Prompt Order` section in this doc somewhere--have you edited that to include this module?

Reply:
I think I missed that one, will fix that ^-^